backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -114,7 +118,8 @@ func (fs *experimentStore) StopExperiments(ctx context.Context, ids []uint64) er
 func (fs *experimentStore) GetExperiments(ctx context.Context) ([]*experimentation.Experiment, error) {
 	sql := `
         SELECT experiment_run.id, details FROM experiment_config, experiment_run
-        WHERE experiment_config.id = experiment_run.experiment_config_id`
+        WHERE experiment_config.id = experiment_run.experiment_config_id
+        AND NOW() <@ experiment_run.execution_time`
```

Comment:
Can we keep this 'generic' GetExperiments method return all experiments (unless the caller specifies otherwise)?